SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -247,7 +246,7 @@ for MODULE in `find %{buildroot}/lib/modules/%{uname_r} -name *.ko` ; do \
 %define __spec_install_post\
     %{?__debug_package:%{__debug_install_post}}\
     %{__arch_install_post}\
-    %{__os_install_post}\
+    %__os_install_post\
```

Comment:
is removing the { } the right call here?

Reply:
I ran the spec linter and that removed it. I can put it back to be consistent with the above line.

Reply:
@jslobodzian this one is a long-standing issue with the spec-linter. We lived with it since the folks changing the kernels were a very limited few who knew the quirks, but we're starting to scale this out, so things like this will start causing more problems.
If we have a backlog for issues about the spec-linter, we should add a note there to tackle when we have time. If not, we probably should start one so we can get the spec-linter into a reliable state.